gradle/libraries.gradle

```diff
@@ -12,7 +12,7 @@ ext {
     junitVersion = '4.12'
     h2Version = '1.4.196'
     bytemanVersion = '4.0.1' //Compatible with JDK10
-    infinispanVersion = '8.2.5.Final'
+    infinispanVersion = '9.2.0.Final'
```

Comment:
If you need the Infinispan upgrade, could you separate that in a specific commit and JIRA ?

Reply:
The WildFly master branch is currently on Infinispan 9.2.0.Final, from what you said before about Infinispan needing to catch up with the ORM 5.3 cache changes, the Infinispan version will likely need to be bumped again.  Should we upgrade to Infinispan 9.2.0.Final for now?

Reply:
I didn't remove the 9.2.0.Final upgrade yet, guess we can do that next but we will be out of sync then with WildFly master.

Reply:
@pferraro can answer better than me, I'm not really sure what JCache configuration will look like in WildFly exactly.  is wildfly-jcache configuration generic for any of the JCache implementations, like EHCache, or is it specific to Infinispan?  I assume it would be specific to Infinispan but not really sure.

Reply:
@scottmarlow For starters, WF would need to provide its own CachingProvider, whose caches are backed by configurations created by the Infinispan subsystem.  Additionally, hibernate-jcache would need to be modified to load its CachingProvider from a specific classloader (e.g. the classloader of the deployment or the classloader of the JCacheRegionFactory class) rather than the thread context classloader (the default behavior of Caching.getCachingProvider(String)).
This is certainly a worthwhile goal - I had proposed a WF JCache provider about 3 years ago, and can resurrect/update that code for use with hibernate-jcache.
However, while this RegionFactory implementation is likely fine for single node, or read-only and non-strict-read-write caches, anyone intending to use read-write caches in a WF cluster will still want to use Infinispan's RegionFactory implementation.